examples/EventV1/EventV2.json

```diff
@@ -0,0 +1,26 @@
+{
+  "specversion": "1.0",
+  "id": "ProjectId:5e1f4591-9868-4663-9f32-524e37ccc1d2",
```

Comment:
I'm currently only using "ProjectId", rather than "ProjectId", "JobId", "TaskId", and "EventId".

Reply:
So we won't know anything about this message other than the project. One of the problems with async systems is tracking where things are coming from and why. The intent of Job, Task, and Event is to help in tracing/debugging, removing those concepts will make that harder.

Reply:
To expand on the above comment. Let's say you send a message from a project and then another message is sent soon after. NAE will process both then post both messages to the portal q. How will the portal know which message is which?

Example: to replay a series of transformations, portal could send 10 messages (1 for each step), which could complete async, so portal will receive back 10 response messages. How will portal know which is which? If you had at least an EventId then you could map that back to the replay Job and the transformation Task, so you could make sense of the responses. I believe that the Job and Task IDs would be helpful in tracing/debugging too.

Reply:
Kevin,

I'm planning on adding some of this additional tracking data just as soon as I get around to building out the infrastructure. As you noticed, right now I'm just generating the message for a single project. This is only to get us through v0.1. I will be expanding this to generate messages for specific variations and then later serializing the message and freezing the contents for each analysis. At that point I'll be able to flesh out the other values you're looking for.